test/Microsoft.AspNetCore.Server.KestrelTests/LibuvTransportTests.cs

```diff
@@ -15,16 +15,14 @@
 using Microsoft.AspNetCore.Server.Kestrel;
 using Microsoft.AspNetCore.Server.Kestrel.Internal;
 using Microsoft.AspNetCore.Server.Kestrel.Internal.Http;
+using Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv;
 using Microsoft.AspNetCore.Testing;
 using Microsoft.Extensions.Internal;
 using Xunit;
 
 namespace Microsoft.AspNetCore.Server.KestrelTests
 {
-    /// <summary>
-    /// Summary description for EngineTests
-    /// </summary>
-    public class EngineTests
+    public class LibuvTransportTests
```

Comment:
Split out the things that are just testing the Libuv transport into the "LibuvTransportTests" (e.g. TransportCanStartAndStop, ListenerCanCreateAndDispose, etc...)

Tests like Http10RequestReceivesHttp11Response don't really belong in "LibuvTransportTests."